backend/.env.example

```
@@ -66,4 +66,5 @@ ADMIN_KEY="random string for admin notice"
 # =================
 # Firebase Admin SDK (FCM 푸시 알림 발송)
 # =================
-FIREBASE_SERVICE_ACCOUNT_PATH=./firebase-adminsdk.json
+base64 < backend/secrets/파일명 | tr -d '\n'
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The base64 < backend/secrets/파일명 | tr -d '\n' key has incorrect delimiter

(IncorrectDelimiter)